com.brave.Brave.metainfo.xml

```diff
@@ -0,0 +1,36 @@
+<!-- Copyright 2017 the Chromium authors, with modifications -->
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.brave.Brave</id>
+  <launchable type="desktop-id">com.brave.Brave.desktop</launchable>
+  <name>Brave Browser</name>
+  <developer_name>Brave Software</developer_name>
+  <summary>Community-maintained unofficial Flatpak of Brave Browser</summary>
```

Comment:
I don't see why summary should say "community-maintained".

Reply:
It was a suggestion from Brave. They said they wanted it to say that it's "community-maintained" and that it's not recommended by Brave themselves

Reply:
Is the new version better?